### PR TITLE
Add connector (api_masterinspcharacteristic_srv)

### PR DESCRIPTION
**Pull request by EsalaMapa**
## Purpose

Fixes:$subject

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: ballerina/api_masterinspcharacteristic_srv/docs.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "description": "The `ballerinax/sap.s4hana.api_masterinspcharacteristic_srv` package offers APIs for seamless integration with the [Master Inspection Characteristic - Read API v2.0.0](https://api.sap.com/api/API_MASTERINSPCHARACTERISTIC_SRV/overview).A master inspection characteristic is a master data record that describes the quality inspection criteria for materials. This service enables you to retrieve, for example, the settings and the control indicators defined for a master inspection characteristic.",
```
Comment (niveathika) [inline ballerina/api_masterinspcharacteristic_srv/docs.json]:
```suggestion
  "description": "The `ballerinax/sap.s4hana.api_masterinspcharacteristic_srv` package offers APIs for seamless integration with the [Master Inspection Characteristic - Read API v2.0.0](https://api.sap.com/api/API_MASTERINSPCHARACTERISTIC_SRV/overview). This service allows to. read quality inspection specifications of the type master inspection characteristic. A master inspection characteristic is a master data record that describes the quality inspection criteria for materials.",
```

File: gradle.properties [hunk 2]
```diff
@@ -5,6 +5,7 @@ group=io.ballerina.lib
 version=1.0.0-SNAPSHOT
 api_inspectionlot_srvVersion=1.0.0-SNAPSHOT
 api_inspectionplan_srvVersion=1.0.0-SNAPSHOT
+api_masterinspcharacteristic_srvVersion =1.0.0-SNAPSHOT
```
Comment (niveathika) [inline gradle.properties]:
```suggestion
api_masterinspcharacteristic_srvVersion=1.0.0-SNAPSHOT
```

File: ballerina/api_masterinspcharacteristic_srv/docs.json (hunk 1 above)
Comment (EsalaMapa) [reply ballerina/api_masterinspcharacteristic_srv/docs.json]: Updated here [Update docs.json and gradle.properties](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/14/commits/b8c8fea8b5fc15e6761bed3bb7808449a7c7443d)

File: gradle.properties (hunk 2 above)
Comment (EsalaMapa) [reply gradle.properties]: Updated here [Update docs.json and gradle.properties](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/14/commits/b8c8fea8b5fc15e6761bed3bb7808449a7c7443d)

**Review by niveathika — APPROVED** [review]

**Pull request merged**